web/lib/api/v1/[word].js

```diff
@@ -0,0 +1,66 @@
+import { words as db } from 'lib/db'
+
+export default function (req, res) {
+  const { word: kw } = req.query
+
+  const matched = db[kw] || db[kw.replaceAll('-', ' ')]
+
+  if (matched !== undefined || matched !== null) {
+    const [first, ...rest] = matched.split('<br>')
+    let word, ipa, ipaVariations
+
+    if (/^@(.*?)(\/.*?\/)(\s+\.*\\s+\/.*\/)?/gms.test(first)) {
+      let parsed = [...first.matchAll(/^@(.*?)(\/.*?\/)(\s+\(.*\)\s+\/.*\/)?/g)]
+      word = parsed[1]
+      ipa = parsed[2]
+      ipaVariations = parsed[3] || null
+    } else {
+      let parsed = [...first.matchAll(/^@(.*?)/gms)]
+      word = parsed[1]
+    }
```

Comment:
Okay so there are words that don't have ipa 🤔

Reply:
yep, and this is a very small sample